rules/tournaments/aoe2de.md

```diff
@@ -0,0 +1,128 @@
+# Age of Empires 2
+
+## General Provisions
+
+These rules inherit all rules from the main rulebook.
+
+These rules apply to all participants of the "Age of Empires 2" tournament (hereinafter "AoE2" or "Game").
+
+The rules may be available in several languages. If the individual versions differ, the English version is considered the source of the truth.
+
+## Tournament Information
+
+### Tournament Platform
+
+The official tournament platform for this tournament is Battlefy.
+Please make sure that you have an account: https://battlefy.com/account/signup
+
+The official tournament page will be shared via the main communication channel established in the main rulebook.
+
+### Tournament Format
+
+The tournament is played in the following stages:
+
+| Stage   | Type               | Format    | Description                                                            |
+|---------|--------------------|-----------|------------------------------------------------------------------------|
+| Stage 1 | Swiss Rounds       | Bo1       | This stage will be seeded according to current 1v1 RM ELO rankings.    |
+| Stage 2 | Single Elimination | Bo1 / Bo3 | The bracket will be seeded according to the result of the first stage. |
+
+### Qualification Swiss System
+
+4 rounds are played according to the Swiss-System. The games will be reallocated after each round according to the
```

Comment:
I would suggest to rewrite it in the follow way:

> In the Swiss-System tournament, there will be a total of four rounds played. After each round, game pairings will be adjusted based on the current player rankings. These rankings are determined by considering the number of games won and lost by each player. Once the four rounds are completed, the top eight players in the rankings will advance to compete for the tournament victory. In the event of a tie for the eighth place after the four rounds, the ranking will be automatically calculated to determine the final eight participants.

Or potentially drop the entire section and link to the [Wikipedia page](https://en.wikipedia.org/wiki/Swiss-system_tournament).